applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/images/mcuboot/prj_release.conf

```diff
@@ -6,9 +6,6 @@
 
 CONFIG_SIZE_OPTIMIZATIONS=y
```

Comment:
Can we optimize the `configurations/nrf52840dk_nrf52840/images/mcuboot/prj_mcuboot_qspi.conf` a bit and enable HW stack protection there too ? (I guess it should not be difficult, I can help if needed :) )

Reply:
fixed